docs/concepts.rst

```diff
@@ -116,6 +116,47 @@ DAGs can be used as context managers to automatically assign new operators to th
 
     op.dag is dag # True
 
+.. _concepts:functional_dags:
+
+Functional DAGs
+---------------
+*Added in Airflow 1.10.11*
```

Comment:
```suggestion
```

Let's not put this in until we're _sure_ it's going to make it in to the release. Sorry.

I would _love_ to have this in 1.10.11, but it might not make it in. We can add this in a separate PR once we've know for sure if it makes it in or not